x-pack/plugins/timelines/public/components/t_grid/toolbar/bulk_actions/alert_status_bulk_actions.tsx

```diff
@@ -91,7 +91,7 @@ export const AlertStatusBulkActionsComponent = React.memo<StatefulAlertStatusBul
             text: i18n.UPDATE_ALERT_STATUS_FAILED_DETAILED(updated, conflicts),
           });
         } else {
-          let title: string;
+          let title;
```

Comment:
Do we need a case for `acknowledged` for the bulk actions here as well?

Reply:
Imo yes. However this pr https://github.com/elastic/kibana/pull/108698 moved this logic into a shared hook https://github.com/elastic/kibana/blob/master/x-pack/plugins/timelines/public/hooks/use_status_bulk_action_items.tsx and this pr adds support for the 4th case https://github.com/elastic/kibana/pull/107972/files#diff-272d1db62d08cfe1d208b7089f9e312a3afa93702adfd7a0792b9eaca8d6bf5aR74 so once you merge, don't have to worry about it here, can just make use of the hook